### ISSUE TITLE
Get the Gaussians and refine it

### ISSUE THREAD

**Issue by Lijiaxin0111** [WU-CVGL/DreamMesh4D#5]
Wonderful Work! Congratulations!

I use the aurorus in Consistent4D to test the model.
When I run 
```
data_root=/dataset/Consistent4D/CONSISTENT4D/input
tag=aurorus
image_path=$data_root/$tag/0.png
mesh_path=./outputs/zero123-sai/aurorus@20250305-111725/simplified_mesh/model_128_972.ply 
python launch.py --config custom/threestudio-dreammesh4d/configs/sugar_static_refine.yaml --train data.image_path=$image_path system.geometry.surface_mesh_to_bind_path=$mesh_path
```

I find the loss decreases in the first 200 steps. However, the loss increased after 200 steps, which confused me.
 Epoch 0: |                                                                            | 200/? [01:08<00:00,  2.91it/s, train/loss=1.070
[INFO] Quantile 0.1: 0.7665
[INFO] Quantile 0.2: 0.8471
[INFO] Quantile 0.3: 0.8753
[INFO] Quantile 0.4: 0.8928
[INFO] Quantile 0.5: 0.9084
[INFO] Quantile 0.6: 0.9212
[INFO] Quantile 0.7: 0.9346
[INFO] Quantile 0.8: 0.9484
[INFO] Quantile 0.9: 0.9645
INFO] Quantile 0.0: 0.003900%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 30/30 [00:05<00:00,  5.55it/s]
[INFO] Quantile 0.1: 0.6425
[INFO] Quantile 0.2: 0.8124
[INFO] Quantile 0.3: 0.8632
[INFO] Quantile 0.4: 0.8907
[INFO] Quantile 0.5: 0.9091
[INFO] Quantile 0.6: 0.9249
[INFO] Quantile 0.7: 0.9408
[INFO] Quantile 0.8: 0.9566
[INFO] Quantile 0.9: 0.9734
Epoch 0: |                                                                                                                                                                                                            | 600/? [03:40<00:00,  2.72it/s, train/loss=84.60[INFO] Quantile 0.0: 0.003900%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 30/30 [00:04<00:00,  6.00it/s]
[INFO] Quantile 0.1: 0.5901
[INFO] Quantile 0.2: 0.7988
[INFO] Quantile 0.3: 0.8582
[INFO] Quantile 0.4: 0.8906
[INFO] Quantile 0.5: 0.9115
[INFO] Quantile 0.6: 0.9291
[INFO] Quantile 0.7: 0.9456
[INFO] Quantile 0.8: 0.9610
[INFO] Quantile 0.9: 0.9773
Epoch 0: |                                                                                                                                                                                                            | 800/? [04:57<00:00,  2.69it/s, train/loss=87.30[INFO] Quantile 0.0: 0.003900%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 30/30 [00:04<00:00,  6.31it/s]
[INFO] Quantile 0.1: 0.5523
[INFO] Quantile 0.2: 0.7883
[INFO] Quantile 0.3: 0.8533
[INFO] Quantile 0.4: 0.8898
[INFO] Quantile 0.5: 0.9128
[INFO] Quantile 0.6: 0.9313
[INFO] Quantile 0.7: 0.9484
[INFO] Quantile 0.8: 0.9644
[INFO] Quantile 0.9: 0.9801

https://github.com/user-attachments/assets/7eee47f0-0f0f-421b-bc3d-f9b667491eab
https://github.com/user-attachments/assets/0997ba2b-2f5e-4f74-9979-1ca728b3fdf0
https://github.com/user-attachments/assets/e5e48e68-091f-4cea-8390-e45da69680c5

**Comment by Lijiaxin0111** [comment]
Video (1) is it200-val.mp4, Video (2) is it400-val.mp4, Video (3) is it1000-val.mp4.

**Comment by Lijiaxin0111** [comment]
Besides, I set the simplify_scale=128 in Simplify the coarse mesh. 
I am not sure What this parameter should be when evaluating in Consistent4D.

**Comment by lizhiqi49** [comment]
Hi, It seems that there is something wrong with the surface Gaussians' scales. Could you try a smaller lr for Gaussian scales?

Actually the configuration is not fixed for different testing samples, and we have to try different hyperparameters for each case, which is the nature of SDS-based optimization... And we found that Gaussians are especially sensitive to lr under SDS.

**Comment by Lijiaxin0111** [comment]
Is it possible to share the hyperparameters for those cases in Consistent4D? We would like to try reproducing the excellent result.

**Comment by lizhiqi49** [comment]
Sorry for late reply, i was occupied by another ddl these days... I will update the configurations files of some cases later.

**Comment by Vxlimo** [comment]
Sorry to bother you. We just want to remind you to update some of the configurations since we really need them. Totally understand if  you are busy.

**Comment by lizhiqi49** [comment]
Sure, I will work on it before this weekend.

**Comment by lizhiqi49** [comment]
@Lijiaxin0111 Hi, I guess you conducted the aurorus experiment with the first frame (0.png) as the reference image. I try to reproduce your issue and find that the extracted mesh with 0.png as refrence image has many floater vertices in the air. And the floaters could be the main reason resulting in your problem.

<img width="471" alt="Image" src="https://github.com/user-attachments/assets/5390fd0a-f8dd-47d2-a8da-dd6a089e3fe2" />

I recommand you to pick a good reference image (e.g., no occlusion for the body parts) as the start of optimization. For the aurorus case, I rerun the static stage with 21.png as reference image which is also utilized in our main paper results, and it works well under the current repository config files. Here is the video it2000-val.mp4:

https://github.com/user-attachments/assets/5c2d1e93-fb39-4aba-a07e-d5d69ad4ad97
